internal/xsql/tx.go

```diff
@@ -197,7 +203,7 @@ func (tx *tx) ExecContext(ctx context.Context, query string, args []driver.Named
 func (tx *tx) PrepareContext(ctx context.Context, query string) (_ driver.Stmt, finalErr error) {
 	onDone := trace.DatabaseSQLOnTxPrepare(tx.conn.trace, &ctx,
 		stack.FunctionID("github.com/ydb-platform/ydb-go-sdk/3/internal/xsql.(*tx).PrepareContext"),
-		&tx.txCtx, tx, query,
+		tx.ctx, tx, query,
```

Comment:
Why you change trace type from pointer to copy of context?

Reply:
I dont want to replace start tx context from any trace callbacks
tx.ctx - only for store some info such as parent trace id